mysql-exporter/mysql-exporter.libsonnet

```diff
@@ -1,40 +1,40 @@
 local k = import 'ksonnet-util/kausal.libsonnet';
 local container = k.core.v1.container;
-local service = k.core.v1.service;
 local containerPort = k.core.v1.containerPort;
 local deployment = k.apps.v1.deployment;
 
-{
+local secrets = import 'secrets.libsonnet';
+
+
+secrets {
+  image:: 'prom/mysqld-exporter:v0.12.1',
+
   _config:: {
     mysql_user: error 'must specify mysql user',
-    mysql_password: error 'must specify mysql password',
-    deployment_name: error 'must specify deployment name',
-    namespace: error 'must specify namespace',
+    mysql_password: '',
```

Comment:
You can put the error back here and below in case you want to require it.

Reply:
I don't think we want to require `mysql_password`. If the user uses the `withSecretPassword` mixin, they won't need a password.

Reply:
If one is using the mixin, it would not touch the error as it does lazy loading.